MiniApp/Classes/Environment.swift

```diff
@@ -44,7 +44,6 @@ internal class Environment {
 
     var baseUrl: URL? {
         guard let endpointUrlString = (self.customUrl ?? bundle.value(for: Key.endpoint.rawValue)) else {
-            Logger.e("Ensure RMAAPIEndpoint value in plist is valid")
```

Comment:
Should we really delete this? It seems it would be useful for developers. Because the error that's returned when they try to create a Mini App won't be very helpful because it won't mention anything about RMAAPIEndpoint, right?